.eslintrc.js

```diff
@@ -1,9 +1,9 @@
 module.exports = {
```

Comment:
@haardikk21 Let's make it right from the start. Let's do it as pure ESM. So, every config is JSON, we use ES2020, do imports with a js extension like './file.js`.

Reply:
Not clear on what the `do imports with a js extension` part means, but switching this to a JSON config.